homework3/semi-uber/Server/api/routes/login.router.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const router = express.Router();
+const config = require("config");
+const { secret } = config.get("JWT");
+const bcrypt = require("bcrypt");
+const { User } = require("../Schemas/user.schema");
+const jwt = require("jsonwebtoken");
+const log4js = require("log4js");
+const logger = log4js.getLogger();
+
+router.post("/", async (req, res) => {
+  try {
+    const { password, username } = req.body;
+    const user = await User.findOne({ username });
+    const token = await jwt.sign(user.id, secret);
+    const verify = await bcrypt.compare(password, user.password);
+
+    if (!verify) {
+      throw new Error();
```

Comment:
again, do not trow errors everywhere just send the user appropriate message and status code and that will be enough

Reply:
Make it to get it catch, if password is not valid and then send response with status